src/TodoForm.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import useInputState from "./useInputState";
+import "./styles.css";
+
+const TodoForm = ({ saveTodo }) => {
+  const { value, reset, onChange } = useInputState();
+
+  return (
+    <form
+      onSubmit={(event) => {
+        event.preventDefault();
```

Comment:
Do you know what this does?

Reply:
delete the input

Reply:
Not really, take a look again about `.preventDefault`
https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault

Reply:
it stops the default action